### PR TITLE
(SIMP-2569) Wrap listsep in a boolean that defaults to true

### PR DESCRIPTION
**Pull request by heliocentric**
SIMP-2569 #close

### CODE REVIEW

**Comment by jeannegreulich** [comment]
Don't push this through yet.  Trevor figured out that the listsep is not working because we put spaces in the access files and when you change the separator, it then counts spaces as part of the name even they are in front.  So we need to remove the spaces from the front.  but he wants to keep this and remove the spaces from the file.

File: manifests/auth.pp [hunk 1]
```diff
@@ -68,6 +68,7 @@
   Boolean          $sssd                      = $::pam::sssd,
   Array[String]    $tty_audit_users           = $::pam::tty_audit_users,
   String           $separator                 = $::pam::separator,
+  Boolean           $enable_separator          = $::pam::enable_separator,
```
Comment (jeefberkey) [inline manifests/auth.pp]: Alignment

File: manifests/init.pp [hunk 2]
```diff
@@ -229,6 +234,7 @@
   Boolean          $use_netgroups             = false,
   Boolean          $use_openshift             = false,
   Boolean          $sssd                      = simplib::lookup('simp_options::sssd', { 'default_value' => false}),
+  Boolean          $enable_separator          = true,
```
Comment (jeefberkey) [inline manifests/init.pp]: This is getting set to `true` now, @jeannegreulich ?

File: metadata.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-pam",
-  "version": "6.0.0",
+  "version": "6.0.1",
```
Comment (jeefberkey) [inline metadata.json]: There hasn't been a release yet

File: manifests/init.pp (hunk 2 above)
Comment (jeannegreulich) [inline manifests/init.pp]: Yes, we want it set to  true by default users are able to use spaces in groups and names.  I explained earlier how we found that the problem was with our adding spaces and Trevor did not want to remove the code Dylan put in to make the list separator optional, so I defaulted it to true and removed the spaces from our rule add.... then I updated all the tests.

**Comment by jeannegreulich** [comment]
I removed my "request changes" (which looks like I approved my own code)  and made the changes Nick Miller asked for.

File: manifests/init.pp (hunk 2 above)
Comment (jeefberkey) [inline manifests/init.pp]: Can you update the PR and the ticket with those details?

**Pull request merged**